ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeer.java

```diff
@@ -425,7 +425,7 @@ Optional<RequestManager> dispatch(final EthMessage ethMessage, final String prot
         localRequestManager -> localRequestManager.dispatchResponse(ethMessage),
         () -> {
           LOG.trace(
-              "Message {} not expected has just been received for protocol {}, peer {} ",
+              "Request message has just been received for protocol {}, peer {} ",
```

Comment:
you're missing a set of {}

Reply:
Thanks @macfarla, just fixed and moved the PR out of draft.